src/k8s-engine/wsl.ts

```diff
@@ -884,6 +884,13 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     await this.wslInstall(trivyExecPath, '/usr/local/bin');
   }
 
+  protected async installHostResolver() {
```

Comment:
This is fine since we are doing [this](https://github.com/rancher-sandbox/rancher-desktop/blob/b65193d08ccdb2d12bb95e6bd81945c9cf45da68/src/assets/scripts/service-host-resolver.initd#L29) but I'm wondering why didn't [this](https://github.com/rancher-sandbox/rancher-desktop/blob/b65193d08ccdb2d12bb95e6bd81945c9cf45da68/src/k8s-engine/wsl.ts#L1265) path work for you? 🤔

Reply:
oh, I'm sorry. Didn't mean to include this change. This was a try to check if I can run host-resolver ok. The original path contains a space in its name. Didn't help. Reverting this change.